src/web/mjs/connectors/templates/FlatManga.mjs

```diff
@@ -58,7 +58,11 @@ export default class FlatManga extends Connector {
             }
             let title = (this.queryChapterTitle ? element.querySelector(this.queryChapterTitle) : element).textContent.replace(manga.title, '');
             let mangaTitle = manga.title.replace(/\s*-\s*RAW$/, '');
-            title = title.replace(new RegExp(mangaTitle, 'i'), '');
```

Comment:
how about `new RegExp(myregex.replace(/[#-}]/g, '\\$&'))` to sanitize the string ?

Reply:
did you test it well first ? I'm not sure if it covers all use cases

Reply:
Hum, if i force the manga title to be in chapter name (for testing), and in use the regex fixed, nothing is removed :/
```
            title += ' '+ mangaTitle;
            title = title.replace(new RegExp(mangaTitle.replace(/[#-}]/g, '\\$&')), '');
```
![image](https://user-images.githubusercontent.com/10671518/208297810-b818c851-22bb-4aec-a428-bb2ad5bbf369.png)

Gotta dig a bit

Reply:
OK got a better regex using this as reference https://www.debuggex.com/cheatsheet/regex/javascript

`            title = title.replace(new RegExp(mangaTitle.replace(/[\*\^\$\.\|\?\+\-\(\)\[\]\{\}]/g, '\\$&')), '');`

Seems to work fine

Reply:
Using this as reference : https://exlskills.com/learn-en/courses/javascript-fundamentals-basics_javascript/regular-expressions-cWSVBpppzSat/applications-lPjEnTmnZThz/special-characters-in-regex-xMIVQGkIObNm

`[\*\^\$\.\|\?\+\-\(\)\[\]\{\}]` seems to work fine